Doesn't work with organization repos?

It doesn't feel like it's working:

```yaml
      - name: Get latest release of rust-analyzer
        uses: rez0n/actions-github-release@v2.0
        id: latest_ra
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          repository: 'rust-lang/rust-analyzer'
          type: stable
```

I'm getting:

```
  File "/entrypoint.py", line 13, in <module>
    repo = G.get_repo(repository)
  File "/usr/local/lib/python3.10/site-packages/github/MainClass.py", line 375, in get_repo
    assert isinstance(full_name_or_id, (str, int)), full_name_or_id
AssertionError: None
```

So it looks like the "repository" input is not actually passed to the python script.

According to [github documentation](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action?learn=create_actions&learnProduct=actions#creating-an-action-metadata-file) it appears that it's necessary to pass inputs as `args` in `action.yml` but I don't see that being done [here](https://github.com/rez0n/actions-github-release/blob/794c12f5e8d629e6ca329cf2e2daeb0f0ce6a3ce/action.yaml#L28-L30).

Same.

```
/usr/bin/docker run --name ee319e5fcdc2d9d0b4dea906f8c7a18c9eae5_beeacb --label 0ee319 --workdir /github/workspace --rm -e "REGISTRY" -e "IMAGE_NAME" -e "DOCKER_METADATA_OUTPUT_VERSION" -e "DOCKER_METADATA_OUTPUT_TAGS" -e "DOCKER_METADATA_OUTPUT_LABELS" -e "DOCKER_METADATA_OUTPUT_JSON" -e "DOCKER_METADATA_OUTPUT_BAKE_FILE" -e "INPUT_TOKEN" -e "INPUT_REPOSITORY" -e "INPUT_TYPE" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKFLOW_REF" -e "GITHUB_WORKFLOW_SHA" -e "GITHUB_WORKSPACE" -e "GITHUB_
Traceback (most recent call last):
  File "/entrypoint.py", line 13, in <module>
    repo = G.get_repo(repository)
  File "/usr/local/lib/python3.10/site-packages/github/MainClass.py", line 4[22](https://github.com/openziti/ziti-builder/actions/runs/7934248772/job/21664804515?pr=31#step:8:23), in get_repo
    assert isinstance(full_name_or_id, (str, int)), full_name_or_id
AssertionError: None
```

--- a/action.yaml
+++ b/action.yaml
@@ -28,3 +28,7 @@ outputs:
 runs:
   using: 'docker'
   image: 'Dockerfile'
+  args:
+    - ${{ inputs.token }}
+    - ${{ inputs.repository }}
+    - ${{ inputs.type }}

